gslib/commands/mb.py

```diff
@@ -247,6 +255,13 @@ def RunCommand(self):
         elif o == '--pap':
           public_access_prevention = a
           json_only_flags_in_command.append(o)
+        elif o == '--placement':
+          placements = a.split(',')
+          if len(placements) != 2:
+            raise CommandException(
+                'Please specify two regions separated by comma.'
```

Comment:
I wonder if we need to say there's no white space after the comma. Maybe say, "two regions in CSV format"

Reply:
I added "without space" to the message. I was not sure about "CSV format" because I feel someone might confuse that with an actual csv file.